### DISCUSSION TITLE
Can't get LPC usecase working with EVBox LIVO

### DISCUSSION THREAD

**Discussion by rvanlier** [enbility/eebus-go#146] [Q&A] [answered]
Hi,

For the past weeks I have been experimenting with the HEMS example in conjunction with the Controlbox example and an EVBox Livo wall charging unit which is on the same network as my computer.
The HEMS and Controlbox processes talk to eachother properly when run on my computer, HEMS receives consumption and production limits as expected and re-announces them to Controlbox. I have managed to get the Livo to recognise the HEMS process as its energy manager via the EVBox Install app. I can also see that the RemoteSKIConnected method is called with the EVBox Livo SKI, and I can see some handshake messages going back and forth. So far, so good.

However it doesn't seem like the limit is ever communicated to the Livo charger. It has been internally limited to 16A because it has been fitted with a standard Schuko connector for testing purposes, so it should never try to draw more than 3680 kW, but regardless of any limits set by Controlbox anywhere between 10 and 3500 W, it always draws around 1200 W when connecting a vehicle.

Is there something obvious I am missing? Like is there another process I should run, or a usecase implementation (and message handling) I should add to one of them?

I see in the readme that it has been tested with two different wall chargers, so I assume the functionality exists, correct? Or is that regarding basic communication only?

Thank you in advance!

**Comment by DerAndereAndi** [comment]
Hi, without a trace log it is impossible to say what is going on

**Comment by sthelen-enqs** [comment, marked as answer]
@rvanlier am I understanding you correctly that you're hooking the examples up controlbox (eebus-go) -> hems (eebus-go) -> EVBox Livo and then setting a limit in the controlbox? The HEMS example isn't hooked up to forward any limits it receives from the controlbox so that won't work. For anything else, we'd need the trace logs (that should already be enabled in the HEMS, and can be enabled in the controlbox by uncommenting the body of the corresponding functions)

**Reply by rvanlier** [reply]
Ah, that explains why, if it's not programmed to forward any limits. Thank you for responding! Is this a feature we could expect sometime in the future?

**Reply by DerAndereAndi** [reply]
The code in the example folder are simply examples on how to use the API. There are not meant to be fully fleged and implemented projects. Also this repository is not meant to include a full implementation of any of these. But you are able to use the examples to write your own HEMS implementing the features you need.
